### ISSUE TITLE
Dark mode support

### ISSUE THREAD

**Issue by praneetloke** [pulumi/registry#244]
## Hello!

- Vote on this issue by adding a 👍 reaction
- If you want to implement this feature, comment to let us know (we'll work with you on design, scheduling, etc.)

## Issue details

When I am browsing the API docs for a package and even simply browsing around in the registry the white background is tough on my eyes. Adding dark mode support would help immensely for late-night hackers like me and also for those who prefer a dark theme at all times.

### Affected area/feature

The whole registry site

**Comment by damusix** [comment]
plllleeeeease <3

**Comment by dgsauve** [comment]
Agreed, went to look for it today and was disappointed when I discovered no such option exists. I would be very grateful for this addition :)

**Comment by susanev** [comment]
we are working on this right now!! you should see this available by the end of the month if not earlier ❤️

**Comment by dgsauve** [comment]
Rad!!

**Comment by dgsauve** [comment]
Middle of May bump ;)

**Comment by susanev** [comment]
sry yall, this was delayed to prioritize our may 4th launch. we ar hoping to pick it back up soon.
https://www.pulumi.com/blog/pulumi-universal-iac/

**Comment by Wittionary** [comment]
Hello platypuddians,
I would also like dark theme. Godspeed.

**Comment by twoTimesAgnew** [comment]
any update on this @susanev ?

**Comment by dgsauve** [comment]
@twoTimesAgnew I've had pretty good luck using the Dark Reader extension to view their site in a dark mode. However, native support is always better.

**Comment by susanev** [comment]
hi! we are doing q4 planning, and this is on my list, so we will be trying super hard to make this happen before the end of the year. 🤞🏻

**Comment by anita-trimbur** [comment]
hi all, the ux team would love to make a dark theme available for pulumi's marketing website, docs, and registry sometime before the year wraps. the design work has been ready for awhile now, so we're hopeful to make space for development soon! leaving a small preview of the homepage with the dark theme enabled. 

<img width="1394" alt="Screen Shot 2022-10-03 at 10 21 57 AM" src="https://user-images.githubusercontent.com/97129308/193606844-d1bc3704-5bb6-473e-8743-7b0c8091078f.png">

a few other things to expect:

* the current light theme will remain default, but you'll be able to turn on dark mode from any page on our marketing website, docs, or registry
* we'll remember your selection so you'll only have to toggle the theme once
* if you have any feedback, please don't hesitate to reach out to us!

**Comment by Wittionary** [comment]
@anita-trimbur Woo hoo! Could you share an example dark-modized paragraph and code block, please? :)

**Comment by anita-trimbur** [comment]
@Wittionary absolutely! here's a preview of the dark theme applied to a package page in the registry

<img width="400" alt="dark mode (1)" src="https://user-images.githubusercontent.com/97129308/193915610-3b6aaf54-3121-4dc7-b947-050a1345d054.png">

**Comment by luillyfe** [comment]
Waiting for you guys!

**Comment by excalq** [comment]
Guessing there are a lot of busy devops folk like myself Pulumi as an extracurricular activity given current industry trends. Dark mode docs would be excellent!

**Comment by dcrockwell** [comment]
:+1:

**Comment by XenoWad01** [comment]
Please? The light burns my eyes :(

**Comment by Wittionary** [comment]
This needs to be reopened.
While [pulumi/docs #19750](https://github.com/pulumi/docs/pull/19750) added dark mode for `/docs/*`, the registry still does not yet have a dark mode.
- [Registry home](https://www.pulumi.com/registry/)
- [Azure Native sqlserver example](https://www.pulumi.com/registry/packages/azure-native/api-docs/azuredata/sqlserver/)

**Comment by AnuiDolphin** [comment]
Agreed that the Registry needs dark mode. I created a separate ticket: https://github.com/pulumi/registry/issues/244 to track that work.
